### ISSUE TITLE
Create simplified credits template for hosts of community-led projects

### ISSUE THREAD

**Issue by Ranzha** [virtual-vgo/vvgo#316]
It's been really tough dealing with text-based credits. Spreadsheets are the future! But I fear that our current submission records sheet is tough to just jump into. Is there a way to keep the current spreadsheet architecture for credits, but hide/obfuscate columns that might not prove useful?

**Comment by MatteuSan** [comment]
Can you provide a sample spreadsheet?

**Comment by Ranzha** [comment]
Yes, that's the plan. It'll be identical in function to the Production Team's submission records sheet, but without some unnecessary elements.

The credits really boil down to (1) credited name, (2) instrument played, (3) part played/sung. Crewside, it's easiest to reference a recent VVGO release and fill in the blanks.
